packages/nx-plugin/src/generators/app/lib/add-analog-project-config.ts

```diff
@@ -43,7 +43,7 @@ export function addAnalogProjectConfig(
         defaultConfiguration: 'development',
         options: {
           buildTarget: `${projectName}:build`,
-          port: 4200,
+          port: 4205,
```

Comment:
This should remain `4200`

```suggestion
          port: 4200,
```

Reply:
fixed